sycl/unittests/Extensions/DefaultContext.cpp

```diff
@@ -18,6 +18,32 @@
 inline constexpr auto EnableDefaultContextsName =
     "SYCL_ENABLE_DEFAULT_CONTEXTS";
 
+void test_contexts_are_equal(const sycl::platform &Plt1,
+                             const sycl::platform &Plt2) {
+  const sycl::device Dev1 = Plt1.get_devices()[0];
+  const sycl::device Dev2 = Plt2.get_devices()[0];
+
+  sycl::queue Queue1{Dev1};
+  sycl::queue Queue2{Dev2};
+
+  ASSERT_EQ(Queue1.get_context(), Queue2.get_context());
+
+  ASSERT_EQ(Dev1.get_platform().ext_oneapi_get_default_context(),
+            Dev2.get_platform().ext_oneapi_get_default_context());
+}
+
+void test_default_context_disabled(const sycl::platform &Plt) {
+  bool catchException = false;
+  try {
+    (void)Plt.ext_oneapi_get_default_context();
+  } catch (const std::runtime_error &) {
+    catchException = true;
+  }
```

Comment:
Why aren't we using `EXPECT_THROW`?

Reply:
I didn't modify the logic test. Just extracted the common code to the function (BTW, after recent changes even more code can be moved to a common function.

Do you still want me to rewrite it with EXPECT_THROW?

Reply:
Fair enough, I won't insist :)